docs/factored_gaussian.rst

```diff
@@ -0,0 +1,94 @@
+Factoring the Gaussian Filter
+=============================
+
+This section provides background on applying a Gaussian filter with a large filter scale by repeatedly applying Gaussian filters with a smaller filter scale.
+This was not discussed in `Grooms et al. (2021) <https://doi.org/10.1029/2021MS002552>`_, so this section provides extra detail.
+
+The :py:class:`gcm_filters.Filter` class has an argument ``n_iterations`` that will automatically split a Gaussian filter into a sequence of Gaussian filters with smaller scales, each of which is less sensitive to roundoff errors. If a user is encountering instability with the standard Gaussian filter, the user can try setting ``n_iterations`` to an integer greater than 1. In general ``n_iterations`` should be set to the smallest value that avoids numerical instability. The user should input their desired filter scale, and then the code will automatically adjust it to reflect the filter scale of the smaller constituent filters.
```

Comment:
In the sentence above - "adjust it" - I can't understand what **it** refers to. In general this section should try to be clearer and more explicit about what exactly is the recommended procedure for users.